src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -101,6 +101,7 @@
 Error_enabling_Features = 'Error while updating agents for enabling features. Please run \"kubectl get pods -n azure-arc\" to check the pods in case of timeout error. Error: {}'
```

Comment:
If we are not planning to take any other cli changes for the next release, we should update the [HISTORY](https://github.com/Azure/azure-cli-extensions/blob/45873317506bc21d240de721559363b34ec7b6ee/src/connectedk8s/HISTORY.rst#release-history) file as well as the [setup](https://github.com/Azure/azure-cli-extensions/blob/45873317506bc21d240de721559363b34ec7b6ee/src/connectedk8s/setup.py#L20) file with a new version

Reply:
We are not planning to merge this for now

Reply:
no even for the POC, to avoid, we should update the version to 1.2.0 min version and build the .whl file based on that

Reply:
ack